### PR TITLE
Documentation Set correct link to ImagePullSecrets for a SA.

### PR DESCRIPTION
**Pull request by born4new**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind documentation

**What this PR does / why we need it**:

Change a link in the API spec for imagePullSecrets for the service account.

**Special notes for your reviewer**:

I didn't re-generate swagger documentation, didn't know how to, or if I had to.

**Does this PR introduce a user-facing change?**:

```release-note
Change the documentation link for the ImagePullSecrets description inside the ServiceAccount API spec.
```

### CODE REVIEW

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -10590,7 +10590,7 @@
           "type": "boolean"
         },
         "imagePullSecrets": {
-          "description": "ImagePullSecrets is a list of references to secrets in the same namespace to use for pulling any images in pods that reference this ServiceAccount. ImagePullSecrets are distinct from Secrets because Secrets can be mounted in the pod, but ImagePullSecrets are only accessed by the kubelet. More info: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod",
+          "description": "ImagePullSecrets is a list of references to secrets in the same namespace to use for pulling any images in pods that reference this ServiceAccount. ImagePullSecrets are distinct from Secrets because Secrets can be mounted in the pod, but ImagePullSecrets are only accessed by the kubelet. More info: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account",
```
Comment (draveness) [inline api/openapi-spec/swagger.json]: I believe this is a generated swagger JSON file, and you should update the source which derives this file and run the `make update` command. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L3754

Comment (born4new) [reply api/openapi-spec/swagger.json]: Thanks @draveness, will do!

**Review by draveness — COMMENTED** [review]
/priority backlog
/sig documentation
/ok-to-test

**Pull request closed without merging**